### PR TITLE
Fixed assigned issue

### PR DESCRIPTION
**Pull request by ScottC44**
Added test cases for error code 114010

### CODE REVIEW

File: test_log_parse.py [hunk 1]
```diff
@@ -61,6 +61,13 @@ def test_ASA_1_114001(self):
         self.assertTrue(self.df.loc[114001, 'Severity'] == 1)
         self.assertTrue(self.df.loc[114001, 'Text'] == 'Failed to initialize 4GE SSM I/O card (error error_string).')
         self.assertTrue(self.df.loc[114001, 'Error'] == 'error_string')
+        
+    def test_ASA_3_114010(self):
+        # %ASA-3-114010: Failed to set multicast hardware address in 4GE SSM I/O card (error error_string).
+        self.assertTrue(self.df.loc[114010, 'Type'] == 'ASA')
+        self.assertTrue(self.df.loc[114010, 'Severity'] == 3)
+        self.assertTrue(self.df.loc[114010, 'Text'] == '(Primary) Failed to set multicast hardware address in 4GE SSM I/O card (error error_string)')
```
Comment (claesmk) [inline test_log_parse.py:69]: This line is causing your unit-test to fail - it should be:
```
+        self.assertTrue(self.df.loc[114010, 'Text'] == 'Failed to set multicast hardware address in 4GE SSM I/O card (error error_string).')
```

File: test_log_parse.py [hunk 2]
```diff
@@ -61,6 +61,13 @@ def test_ASA_1_114001(self):
...
+        self.assertTrue(self.df.loc[114010, 'Error'] == 'error-string.')
```
Comment (claesmk) [inline test_log_parse.py:70]: This line is also causing a problem - it should be:
```
+        self.assertTrue(self.df.loc[114010, 'Error'] == 'error_string')
```

**Review by claesmk — CHANGES_REQUESTED** [review]
Please see my comments for why your unit-test is failing

**Comment by claesmk** [comment]
Reminder @ScottC44 - you need to fix your unit-test issue before committing

File: test_log_parse.py [hunk 3]
```diff
@@ -61,6 +61,13 @@ def test_ASA_1_114001(self):
         self.assertTrue(self.df.loc[114001, 'Severity'] == 1)
         self.assertTrue(self.df.loc[114001, 'Text'] == 'Failed to initialize 4GE SSM I/O card (error error_string).')
         self.assertTrue(self.df.loc[114001, 'Error'] == 'error_string')
+        
+    def test_ASA_3_114010(self):
+        # %ASA-3-114010: Failed to set multicast hardware address in 4GE SSM I/O card (error error_string).
+        self.assertTrue(self.df.loc[114010, 'Type'] == 'ASA')
+        self.assertTrue(self.df.loc[114010, 'Severity'] == 3)
+        self.assertTrue(self.df.loc[114010, 'Text'] == 'Failed to set multicast hardware address in 4GE SSM I/O card (error error_string).')
+        self.assertTrue(self.df.loc[114010, 'Error'] == 'error_string.')
```
Comment (claesmk) [inline test_log_parse.py:70]: unit-test is still failing - error_string should not have a period after it

Comment (ScottC44) [reply test_log_parse.py]: Sorry for the constant errors. I think I finally fixed everything

**Review by claesmk — APPROVED** [review]

**Pull request merged**